tests/unit/amo/sagas/test_landing.js

```diff
@@ -78,6 +79,7 @@ describe(__filename, () => {
         const baseFilters = {
           addonType: getAddonTypeFilter(addonType),
           page_size: pageSize,
+          recommended: addonType === ADDON_TYPE_EXTENSION ? true : undefined,
```

Comment:
I suppose this sort of cover the case when the add-on is not an extension, but I would like to see a test case for that (to make sure we do not pass this filter when the addon is not an extension, no matter the value of the feature flag)

Reply:
You mean a test where the feature is disabled and the add-on isn't an extension? Because I agree that this test case already covers the case where an add-on is not an extension (and the feature is enabled). For the former, we now have `does not include only recommended add-ons when the enableFeatureRecommendedBadges flag is false` so I'm not sure we also need one where the flag is false *and* the extension isn't an add-on. If you think that is important, then I could add an `it.each()` to the new test case, but I don't think we really need it.

Reply:
@willdurand I'm actually planning to enhance these tests for the patch for #8028, so maybe we can land this one as is for now, and more coverage will be added in that patch (which I'm currently writing).

Reply:
> You mean a test where the feature is disabled and the add-on isn't an extension? Because I agree that this test case already covers the case where an add-on is not an extension (and the feature is enabled). For the former, we now have `does not include only recommended add-ons when the enableFeatureRecommendedBadges flag is false` so I'm not sure we also need one where the flag is false _and_ the extension isn't an add-on. If you think that is important, then I could add an `it.each()` to the new test case, but I don't think we really need it.

Is there a test case for when the feature is enabled but the add-on isn't an extension?

> @willdurand I'm actually planning to enhance these tests for the patch for #8028, so maybe we can land this one as is for now, and more coverage will be added in that patch (which I'm currently writing).

yep

Reply:
> 
> Is there a test case for when the feature is enabled but the add-on isn't an extension?
>

Yes, the test at https://github.com/mozilla/addons-frontend/blob/df12f9830671f3284a0314ef5a02ad67db3af330/tests/unit/amo/sagas/test_landing.js#L75 does that, but the flag being enabled isn't explicit in the test, so I've changed it to make it explicit.

> > @willdurand I'm actually planning to enhance these tests for the patch for #8028, so maybe we can land this one as is for now, and more coverage will be added in that patch (which I'm currently writing).
> 
> yep

Cool, thanks, I'll do that.